src/components/dashboard/Dashboard.js

```diff
@@ -243,6 +245,15 @@ const Dashboard = props => {
     ],
   }
 
+  useEffect(() => {
+    const debouncedHandleResize = debounce(() => {
+      log.info('update component after resize', update)
+      setUpdate(Date.now())
```

Comment:
this does nothing to the screen size, what exactly did you fix?

Reply:
This is an empty set that will allow to re-render the component

Reply:
to fix the problem you need to re-render the component after resize